includes/template-library/sources/local.php

```diff
@@ -1572,6 +1576,9 @@ private function add_actions() {
 			add_action( 'manage_posts_extra_tablenav', [ $this, 'maybe_render_blank_state' ] );
 		}
 
+		// REST API Actions
+		( new REST_API() )->register();
```

Comment:
Maybe move the hook from the `register` to here in order to create an instance only when needed

Reply:
The idea of this class was not to add another hook to the module, the class should be responsible for all the hooks that related to the REST API for the "local" Elementor library

Reply:
Removed